src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedAccessorValueParameterSymbol.cs

```diff
@@ -80,18 +80,22 @@ internal override OneOrMany<SyntaxList<AttributeListSyntax>> GetAttributeDeclara
         internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, ref ArrayBuilder<CSharpAttributeData> attributes)
         {
             base.AddSynthesizedAttributes(moduleBuilder, ref attributes);
+            if (this.ContainingSymbol is SourcePropertyAccessorSymbol { AssociatedSymbol: SourcePropertySymbolBase property })
+            {
+                AddSynthesizedFlowAnalysisAttributes(this, property, ref attributes);
+            }
+        }
 
-            if (ContainingSymbol is SourcePropertyAccessorSymbol propertyAccessor && propertyAccessor.AssociatedSymbol is SourcePropertySymbolBase property)
```

Comment:
>if (ContainingSymbol is SourcePropertyAccessorSymbol propertyAccessor && propertyAccessor.AssociatedSymbol is SourcePropertySymbolBase property) [](http://example.com/codeflow?start=12&length=144)

I think this condition should be part of the extracted helper. Basically I would like to make the same suggestion as for a similar helper in ```SourcePropertyAccessorSymbol``` - make it an instance method by simply extracting the relevant portion as is (the entire `if` statement in this particular case)